app/models/like.rb

```diff
@@ -1,7 +1,12 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_create :update_likes_counter
+
+  private
+
   def update_likes_counter
-    post.update(likes_counter: post.likes.count)
+    post.increment!(:likes_counter)
   end
 end
```

Comment:
- [ ] As required for the project to have tests to all your models, kindly make sure you also add unit tests to both `Like` and `Comment` models as right now they were not implemented yet.

- [ ] You should also ensure that to add corresponding unit tests to all custom methods in each model such as to the `update_likes_counter` and the `update_comments_counter` methods. This is for the project requirements.